spec/controllers/items_controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+describe ItemsController, type: :controller do
+  describe '#index' do
+    subject { get :index }
+
+    let!(:items)  { create_list(:item, rand(4..7)) }
+    let(:expected_list) { items.sort_by(&:brand) }
```

Comment:
what is a different between `sort_by(&:brand)` and `sort(:brand)` ?

Reply:
Because `create_list` create common array of items. And I can't use `sort(:brand) ` which working with ActiveRecord collection.